commitlint/plugins.ts

```diff
@@ -262,6 +262,44 @@ export abstract class Plugins {
         ];
     }
 
+    public static defaultRevertMessage(
+        headerStr: string,
+        bodyStr: string | null,
+        when: string
+    ) {
+        let offence = false;
+        let isRevertCommitMessage = headerStr.toLowerCase().includes("revert");
+
+        const negated = when === "never";
+
+        if (isRevertCommitMessage) {
+            let isDefaultRevertHeader =
+                headerStr.match(/^[Rr]evert ".+"$/) !== null;
+
+            if (isDefaultRevertHeader) {
+                if (bodyStr !== null) {
+                    let lines = bodyStr.split("\n");
+                    offence =
+                        // 40 is the length of git commit hash.
+                        lines.length == 1 &&
+                        lines[0].match(/^This reverts commit [^ ]{40}\.$/) !==
+                            null;
+                } else {
+                    offence = true;
+                }
+            }
+
+            offence = negated ? offence : !offence;
+        }
+        return [
+            !offence,
+            (negated
+                ? `Please explain why you're reverting`
+                : `Please don't change the default revert commit message`) +
+                Helpers.errMessageSuffix,
```

Comment:
@realmarv if you check what is in Helpers.errMessageSuffix you will see that you need to end the previous error messages with a dot; otherwise it will read wrong

Reply:
Done